### PR TITLE
Create render-lineage-feature.md

### PR DESCRIPTION
**Pull request by mirnawong1**
created a release note for large dag rendering per notion request here: https://www.notion.so/dbtlabs/DAG-in-the-IDE-will-not-auto-render-large-DAGs-e654861a099d4b789fb8d36176594c32

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/docs/dbt-cloud/release-notes/07-July-2022/render-lineage-feature.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Enhancement: Large DAG feature"
+id: "render-lineage-feature"
+description: "Add a render button for Large DAGs to visualize"
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/release-notes/07-July-2022/render-lineage-feature.md]:
```suggestion
description: "Add a render button to visualize large DAGs"
```

File: website/docs/docs/dbt-cloud/release-notes/07-July-2022/render-lineage-feature.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+sidebar_label: "Enhancement: Large DAG feature"
+tags: [July-5-2022, v1.1.56]
+
+---
+
+You can now select **Render Lineage** to visualize large DAGs. 
+
+Large DAGs can take a long time (10+ seconds, if not minutes) to render and can cause browsers to crash.
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/release-notes/07-July-2022/render-lineage-feature.md]:
```suggestion
Large DAGs can take a long time (10 or more seconds, if not minutes) to render and can cause browsers to crash. 
```

File: website/docs/docs/dbt-cloud/release-notes/07-July-2022/render-lineage-feature.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+sidebar_label: "Enhancement: Large DAG feature"
+tags: [July-5-2022, v1.1.56]
+
+---
+
+You can now select **Render Lineage** to visualize large DAGs. 
+
+Large DAGs can take a long time (10+ seconds, if not minutes) to render and can cause browsers to crash. 
+
+The new button feature prevents large DAGs from rendering automatically. Instead, you can select **Render Lineage** to load up the visualization. This should affect only ~15% of the DAGs.
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/release-notes/07-July-2022/render-lineage-feature.md]:
```suggestion
The new button prevents large DAGs from rendering automatically. Instead, you can select **Render Lineage** to load the visualization. This should affect about 15% of the DAGs.
```

**Review by nghi-ly — CHANGES_REQUESTED** [review]
looks good! small nits

**Comment by mirnawong1** [comment]
thank you very much @nghi-ly 🔥 !

**Review by mirnawong1 — COMMENTED** [review]
changes made

**Review by nghi-ly — APPROVED** [review]
:shipit:

**Pull request merged**